src/MassTransit/Middleware/Outbox/InMemoryOutboxMessageRepository.cs

```diff
@@ -26,16 +27,11 @@ public async Task<InMemoryInboxMessage> Lock(Guid messageId, Guid consumerId, Ca
         {
             var key = new InMemoryInboxMessageKey(messageId, consumerId);
 
-            if (!_dictionary.TryGetValue(key, out var existing))
+            var existing = _dictionary.GetOrAdd(key, (_) => new InMemoryInboxMessage(messageId, consumerId)
```

Comment:
sorry, but we do not use `()` for single function parameter

Reply:
:) done